docs/adopters.md

```diff
@@ -29,6 +31,7 @@
 - [TrueLayer](https://truelayer.com)
 - [ViacomCBS](https://viacomcbs.com)
 - [nais](https://nais.io)
+- [Kong](https://konghq.com)
```

Comment:
oh, very cool. i used kong in my previous job (..in a somewhat convoluted way through shipcat).